cds_ils/ldap/api.py

```diff
@@ -206,7 +211,11 @@ def import_users():
     imported = 0
     importer = LdapUserImporter()
     for ldap_user in ldap_users:
-        email = ldap_user_get(ldap_user, "mail").lower()
+        if "mail" not in ldap_user:
+            print("User with employee ID {} does not have an email"
```

Comment:
maybe not print but log ?

Reply:
I think this is supposed to be run from the command line; but I agree for the one in `import_users`.